### DISCUSSION TITLE
分享一个在Firefox下输入框变大的修复教程

### DISCUSSION THREAD

**Discussion by Xia0xia0Xia0** [F9y4ng/GreasyFork-Scripts#76] [经验分享(Share Experience)]
Firefox浏览器使用字体渲染脚本更改一些比较特殊的字体（我用的是苹方）后，就会出现下面的情况：

![111](https://user-images.githubusercontent.com/101876873/159209918-76defc00-688c-4a41-a19c-3673cbc92797.png)

这时候要修复这个bug，需要以下操作：

打开脚本界面，在需要渲染的网页元素里添加 ```:not(input)```，然后再在排除渲染的HTML标签里删除```input,```，再保存就好啦！

![222](https://user-images.githubusercontent.com/101876873/159210135-1c51249b-7cd3-438b-89ce-fa2ef73c6566.png)

我很强，我是笑小笑。

**Comment by F9y4ng** [comment]
Firefox确实会根据不同字体的字宽来绘制input标签宽度，个别字体会导致input被撑开，将input排除渲染即可。
